src/scripts/readmeScript.ts

```diff
@@ -0,0 +1,36 @@
+import { COMMANDS } from '../commands/commands';
+import { readFile, writeFile } from 'fs';
```

Comment:
Change to 
```ts
import { promises } from 'fs';
const { readFile, writeFile } = promises;
```

this will give you promise versions of the callback methods. Available after node 10 I believe (we are using 12, so it should be good)